src/main/java/com/kryptnostic/api/v1/indexing/PaddedMetadataMapper.java

```diff
@@ -75,17 +68,18 @@ public MappedMetadata mapTokensToKeys( Set<Metadata> metadata, EncryptedSearchSh
 
             if ( metadatumList == null ) {
                 metadatumList = Lists.newArrayListWithExpectedSize( 1 );
-                metadataMap.put( indexForTerm, metadatumList );
+                metadataMap.put( ByteBuffer.wrap( indexForTerm ), metadatumList );
             }
 
             metadatumList.add( balancedMetadatum );
 
         }
-        log.info(
-                "[PROFILE] MinLocations: {} MaxLocations: {} RawMetadataSize: {} ProcessedMetadataSize: {} AcceptedTokens: {}",
-                metadata.size(),
-                metadataMap.values().size(),
-                numAcceptedTokens );
+        log
```

Comment:
We need to get auto-formatter sorted. Something in the current one is forcing this weirdness.